apps/shared/avifutil.c

```diff
@@ -412,7 +412,7 @@ void avifDumpDiagnostics(const avifDiagnostics * diag)
 
 // Windows
 
-#include <windows.h>
+#include <Windows.h>
```

Comment:
Otherwise, we get the warning:
```
D:\a\libavif\libavif\apps\shared\avifutil.c(415,10): error: non-portable path to file '<Windows.h>'; specified path differs in case from file name on disk [-Werror,-Wnonportable-system-include-path]
  415 | #include <windows.h>
      |          ^~~~~~~~~~~
      |          <Windows.h>
```

Reply:
reverted